radar/languages_frameworks/react.yaml

```diff
@@ -2,13 +2,17 @@ version: 1
 name: React
 logo: https://upload.wikimedia.org/wikipedia/commons/a/a7/React-icon.svg
 blip:
-  - version: "2.2"
```

Comment:
Leave the 2.2. update as-is and add the 2.4 update. We want the history to be visible in the radar.